spec/features/users/user_can_see_ghost_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
```

Comment:
This is personal preference and you don't have to change this, but I would have distributed these tests into the specs corresponding to the views (posts/index_spec, profile/posts/show_spec, etc) as opposed to testing them in a separate test - this would allow you to recycle the setup of existing tests.

Also, I'd say it suffices to just `expect(page).to have_css(".ghost-container") since that div contains all of the other pieces.